### PR TITLE
fix panic in snapshot path

### PR DESCRIPTION
**Pull request by ouwenkg**
# Fix panic in snapshot path

选区_037

### Applicable Issues
#496

### CODE REVIEW

File: tools/snapshot_tool/src/main.rs [hunk 1]
```diff
@@ -58,9 +58,7 @@ fn main() {
         .get_matches();
     let command = matches.value_of("cmd").expect("provide specific command");
     let command = cast_command(command);
-    let file = matches
-        .value_of("file")
-        .expect("provice specific snapshot-file path");
+    let file = matches.value_of("file").unwrap_or("snapshot");
```
Comment (leeyr338) [inline tools/snapshot_tool/src/main.rs]: User get error message:
```
output: -bash: ../../bin/snapshot_tool: cannot execute binary file
```
Why this change can fix the bug? Has user run `panic` ?

Comment (ouwenkg) [reply tools/snapshot_tool/src/main.rs]: Noop! 
This change solves `ls snap*` no result problem. Using default file path config, snapshot tool will panic.

`cannot execute binary file` it occured only in mac，caused by different platforms.

Comment (kaikai1024) [reply tools/snapshot_tool/src/main.rs]: So it's nothing about the #496

> cannot execute binary file it occured only in mac，caused by different platforms.

Update the conclusion into the #496.

**Comment by kaikai1024** [comment]
@KaoImin And this commit.
